packages/perseus/src/widgets/sorter/sorter.tsx

```diff
@@ -84,12 +77,9 @@ class Sorter extends React.Component<Props, State> {
         });
     };
 
-    getUserInput: () => {
-        options: any;
-    } = () => {
-        // eslint-disable-next-line react/no-string-refs
+    getUserInput: () => UserInput = () => {
         // @ts-expect-error - TS2339 - Property 'getOptions' does not exist on type 'ReactInstance'.
-        return {options: this.refs.sortable.getOptions()};
+        return {values: this.refs.sortable.getOptions()};
```

Comment:
I don't think we should be changing the names of things where possible. This is an external API and we don't know what it might be used by.

Reply:
Reverted the naming. Thanks! :)